enum/Analytics.ts

```diff
@@ -46,6 +48,14 @@ export const Events: Record<string, IAnalyticsEvent> = {
 			failure_reason: 'network/app error in SF app',
 		},
 	},
+	[EventName.AGENT_TRANSFER_FAILED_DUE_TO_DISCONNECTED_AGENT]: {
+		category: EventCatagory.AGENT_DISCONNECTED,
+		eventType: EventType.SESSION,
+		action: 'triggered',
+		properties: {
+			failure_reason: 'livechat agent disconnnected',
+		},
+	},
```

Comment:
`AGENT_TRANSFER_FAILED_DUE_TO_DISCONNECTED_AGENT` It's not actually Agent Transfer FAILURE, right? As @ear-dev mentioned we are receiving this message when the salesforce agent transfers chat to another salesforce agent